Source/CarthageKit/Xcode.swift

```diff
@@ -984,10 +1021,10 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 					}
 
 			case 2:
-				let firstSDK = sdks[0]
-				let secondSDK = sdks[1]
+				let deviceSDK = sdks[0]
```

Comment:
Then we wouldn't need the `Comprable` implementation which IMO doesn't make much sense for a type like `SDK`.